### ISSUE TITLE
c.k.m.base.reduce.DiffingMPGProcessor#stuffDiffByDiffer() should work quickly when the left Material is null

### ISSUE THREAD

**Issue by kazurayam** [kazurayam/materialstore#416]
derived from

- https://github.com/kazurayam/VisualInspectionOfExcelAndPDF/issues/14

```
2023-04-30 08:56:13.592 WARN  c.k.m.base.reduce.DiffingMPGProcessor    - #stuffDiffByDiffer left Material was NULL_OBJECT. right={"jobName":"NISA_Chronos","jobTimestamp":"20230430_085419","id":"0f58b2d68b0e043644b8b1436f4814f87cf8e459","fileType":"csv","metadata":{"seq":{"key":"seq","value":"1","matchedByAster":true}, "sheet_index":{"key":"sheet_index","value":"0","matchedByAster":true}, "sheet_name":{"key":"sheet_name","value":"指定インデックス投資信託","matchedByAster":true}, "URL.host":{"key":"URL.host","value":"www.fsa.go.jp","matchedByAster":true}, "URL.path":{"key":"URL.path","value":"/policy/nisa/20170614-2/23.xlsx","matchedByAster":true}, "URL.port":{"key":"URL.port","value":"80","matchedByAster":true}, "URL.protocol":{"key":"URL.protocol","value":"https","matchedByAster":true}},"identification":"{\"seq\":\"1\", \"sheet_index\":\"0\", \"sheet_name\":\"指定インデックス投資信託\", \"URL.host\":\"www.fsa.go.jp\", \"URL.path\":\"/policy/nisa/20170614-2/23.xlsx\", \"URL.port\":\"80\", \"URL.protocol\":\"https\"}","relativeUrl":"NISA_Chronos/20230430_085419/objects/0f58b2d68b0e043644b8b1436f4814f87cf8e459.csv","diffability":"AS_TEXT"}
2023-04-30 08:56:15.221 WARN  c.k.m.base.reduce.DiffingMPGProcessor    - #stuffDiffByDiffer left Material was NULL_OBJECT. right={"jobName":"NISA_Chronos","jobTimestamp":"20230430_085419","id":"a057919e32886466213e5393068d27f39bd48f5f","fileType":"csv","metadata":{"seq":{"key":"seq","value":"1","matchedByAster":true}, "sheet_index":{"key":"sheet_index","value":"1","matchedByAster":true}, "sheet_name":{"key":"sheet_name","value":"指定インデックス投資信託以外の投資信託（アクティブ運用投信等）","matchedByAster":true}, "URL.host":{"key":"URL.host","value":"www.fsa.go.jp","matchedByAster":true}, "URL.path":{"key":"URL.path","value":"/policy/nisa/20170614-2/23.xlsx","matchedByAster":true}, "URL.port":{"key":"URL.port","value":"80","matchedByAster":true}, "URL.protocol":{"key":"URL.protocol","value":"https","matchedByAster":true}},"identification":"{\"seq\":\"1\", \"sheet_index\":\"1\", \"sheet_name\":\"指定インデックス投資信託以外の投資信託（アクティブ運用投信等）\", \"URL.host\":\"www.fsa.go.jp\", \"URL.path\":\"/policy/nisa/20170614-2/23.xlsx\", \"URL.port\":\"80\", \"URL.protocol\":\"https\"}","relativeUrl":"NISA_Chronos/20230430_085419/objects/a057919e32886466213e5393068d27f39bd48f5f.csv","diffability":"AS_TEXT"}

```

Why "c.k.m.base.reduce.DiffingMPGProcessor    - #stuffDiffByDiffer" took 2 minuites? Why this long time?

**Comment by kazurayam** [comment]
First I should reproduce the problem in a junit test suite.

**Comment by kazurayam** [comment]
fixed at v0.16.2
